flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -234,6 +271,17 @@ - (void)pendingNotificationRequests:(FlutterResult _Nonnull)result {
   }
 }
 
+- (void)getActiveNotifications:(FlutterResult _Nonnull)result {
+  if (@available(iOS 10.0, *)) {
+    [self activeUserNotificationRequests:result];
+  } else {
+    result([FlutterError
+      errorWithCode:GET_ACTIVE_NOTIFICATIONS_ERROR_CODE
+            message:GET_ACTIVE_NOTIFICATIONS_ERROR_MESSAGE
+            details:[NSNull null]]);
```

Comment:
whilst passing [NSNull null] may work, my understanding is that it's more appropriate to pass `nil` as NSNull is more used for when `nil` is prohibited (ref: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/NumbersandValues/Articles/Null.html) and there's existing examples in the Flutter ecosystem where you find `nil` used e.g. https://github.com/flutter/plugins/blob/b548ae18646d1725a8e1f004c26bf42e4d362aa8/packages/battery/battery/ios/Classes/FLTBatteryPlugin.m#L62